examples/examplepb/BUILD.bazel

```diff
@@ -0,0 +1,51 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+
+package(default_visibility = ["//visibility:public"])
+
+proto_library(
+    name = "examplepb_proto",
+    srcs = [
+        "a_bit_of_everything.proto",
+        "echo_service.proto",
+        "flow_combination.proto",
+        "stream.proto",
+        "wrappers.proto",
+    ],
+    deps = [
+        "//examples/sub:sub_proto",
+        "//examples/sub2:sub2_proto",
+        "//protoc-gen-swagger/options:options_proto",
+        "@com_github_googleapis_googleapis//google/api:api_proto",
```

Comment:
@yugui Any idea on how to have Gazelle to respect that?

Reply:
Sorry, I have no idea. Let me merge this PR as it is.